zerver/fixtures/bugdown-data.json

```diff
@@ -121,6 +121,12 @@
       "bugdown_matches_marked": true
     },
     {
+      "name": "escape",
+      "input": "\\**foo**",
+      "expected_output": "<p>**foo**</p>",
+      "bugdown_matches_marked": true
+    },
```

Comment:
and I didn't get what you mean by changing echo.js implementation.. it looks like these fixtures used with frontend tests, isn't they?